.github/workflows/main.yml

```
@@ -0,0 +1,35 @@
+name: Build
+
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
```

This will make it so it runs once only when a PR is opened, or when we merge to `master`. Right now it'll run two jobs for every PR, slightly wasteful.

We _could_ do just `on: [push]` but building wheels on every single commit seems excessive?